src/utils/sorting-functions.ts

```diff
@@ -0,0 +1,123 @@
+import { ToDoItem, Priority } from '../models/todoitem-model';
+
+export function sortByTitle(items: ToDoItem[], order: number): ToDoItem[] {
+  const orderedItems = items;
+  if (order === 1) {
+    orderedItems.sort((a, b) => {
+      const titleA = a.title.toLowerCase();
+      const titleB = b.title.toLowerCase();
+
+      if (titleA < titleB) {
+        return -1;
+      }
+      if (titleA > titleB) {
+        return 1;
+      }
+      return 0;
+    });
+  } else {
+    orderedItems.sort((a, b) => {
+      const titleA = a.title.toLowerCase();
+      const titleB = b.title.toLowerCase();
+
+      if (titleA > titleB) {
+        return -1;
+      }
+      if (titleA < titleB) {
+        return 1;
+      }
+      return 0;
+    });
+  }
+  return orderedItems;
+}
+
+export function sortByText(items: ToDoItem[], order: number): ToDoItem[] {
+  const orderedItems = items;
+  if (order === 1) {
+    orderedItems.sort((a, b) => {
+      const textA = a.text.toLowerCase();
+      const textB = b.text.toLowerCase();
+
+      if (textA < textB) {
+        return -1;
+      }
+      if (textA > textB) {
+        return 1;
+      }
+      return 0;
+    });
+  } else {
+    orderedItems.sort((a, b) => {
+      const textA = a.text.toLowerCase();
+      const textB = b.text.toLowerCase();
+
+      if (textA > textB) {
+        return -1;
+      }
+      if (textA < textB) {
+        return 1;
+      }
+      return 0;
+    });
+  }
+  return orderedItems;
+}
+
+function isHigherPriority(a: Priority, b: Priority): boolean {
+  if (a === b) {
+    return true;
+  }
+  // a is High and b is not equal to it, so it must be lower
+  if (a === Priority.high) {
```

Comment:
What kind of enum do you have here? Are the values strings? Or numbers? If the enum is just a simple enum the values are represented as numbers and that means you can just directly compare them, without the need for this complex condition